### PR TITLE
Trim the data of the `url` file before filling the `url.src` field

### LINKED ISSUE #190: wrong opam file produced by dune-release opam pkg
Hi,

I'm using [this script](https://git.zapashcanon.fr/zapashcanon/omg/src/branch/master/.build.yml#L72). On the highlighted line, dune-release is producing a wrong opam file.

It's correctly reading the url from the file created on the line before but output something like:

```json
...
url {
  src: """
https://fs.zapashcanon.fr/archive/omg/omg-d8211a1.tbz
"""
...
```

Which doesn't work for opam. It should produce something like:

```json
...
url {
  src: "https://fs.zapashcanon.fr/archive/omg/omg-d8211a1.tbz"
...
```

Currently I've got to fix this by hand and then it's working.

### PR DESCRIPTION
**Pull request by gpetiot**
fix #190

### CODE REVIEW

**Comment by gpetiot** [comment]
Can we merge this one?

File: CHANGES.md [hunk 1]
```diff
@@ -28,6 +28,7 @@
 - Build and run tests for all selected packages when checking distribution tarball
   (#266, @NathanReb)
 - Improve trimming of the changelog to preserve the indentation of the list of changes. (#268, @gpetiot)
+- The data read from the url file must be trimmed before filling url.src field (#270, @gpetiot)
```
Comment (NathanReb) [inline CHANGES.md:31]: Can you improve the changelog entry please, this won't speak to users!

**Review by NathanReb — COMMENTED** [review]
Thanks! It looks good except for the changelog entry!

File: CHANGES.md (hunk 1 above)
Comment (gpetiot) [reply CHANGES.md]: done

**Comment by zapashcanon** [comment]
@gpetiot thanks ! :)

**Pull request merged**